experimental/dds/tree2/src/test/snapshots/summary.spec.ts

```diff
@@ -58,3 +71,74 @@ describe("snapshot tests", () => {
 		});
 	}
 });
+
+const schemaTrees: {
```

Comment:
Can't we just use the test trees from `experimental\dds\tree2\src\test\testTrees.ts`? They expose schemaData directly.